src/components/App/App.component.jsx

```diff
@@ -1,57 +1,61 @@
-import React, { useLayoutEffect } from 'react';
-import { BrowserRouter, Switch, Route } from 'react-router-dom';
-
-import AuthProvider from '../../providers/Auth';
-import HomePage from '../../pages/Home';
-import LoginPage from '../../pages/Login';
-import NotFound from '../../pages/NotFound';
-import SecretPage from '../../pages/Secret';
-import Private from '../Private';
-import Fortune from '../Fortune';
-import Layout from '../Layout';
-import { random } from '../../utils/fns';
+import React from 'react';
+import Navbar from '../Navbar/Navbar.component'
+import ListVideos from '../ListVideos/ListVideos.component';
+import videos from '../../videos.js';
+import VideoDetails from '../VideoDetails/VideoDetails.component'
 
 function App() {
-  useLayoutEffect(() => {
-    const { body } = document;
 
-    function rotateBackground() {
-      const xPercent = random(100);
-      const yPercent = random(100);
-      body.style.setProperty('--bg-position', `${xPercent}% ${yPercent}%`);
+  const [dataVideos, setDataVideos] = React.useState([]); 
+  const [showDetails, setShowDetails] = React.useState(false);
+  const [videoSelected, setVideoSelected] = React.useState('');
+
+  const apiKey = 'AIzaSyBLgZkvZQtw0iEKY9tKCq5oJIPN5WZVmuQ';
+
+  React.useState(()=>{
+
+  },[dataVideos]);
+
+  const hideList = (item) =>{
+    setShowDetails(true); 
+    // console.log('INFO ITEM: ')
+    // console.log(item)
+    setVideoSelected(item);
+  }
+  
+  const dataFromSearch = (e,busqueda) =>{
+    e.preventDefault();    
+    setShowDetails(false); 
+    fetch(`https://www.googleapis.com/youtube/v3/search?part=snippet&&key=${apiKey}&type=video&maxResults=25&q=${busqueda}`)
+         .then(res => {
+             return res.json()
+         })
+         .then(data => {
+             console.log(data); 
+            setDataVideos(data); 
+         }).catch(error=> console.log(error))
+         
+  }
```

Comment:
Is better if you put the fetch call into another file as encapsulation logic, try to do it with a custom hook as part of the bonus 😄